src/Views/DisplaysView.vala

```diff
@@ -110,7 +110,8 @@ public class Display.DisplaysView : Gtk.Grid {
                 mirror_grid.sensitive = monitor_manager.monitors.size > 1;
```

Comment:
I think the mirror switch should be bound to the monitor_manager "is_mirrored" property not the number of monitors?  Not strictly part of this PR though.